### ISSUE TITLE
Wie fülle ich die Datenbanken (mit den ganzen IDs, Locations, ...)?

### ISSUE THREAD

**Issue by Filler-dev** [juztim/Simple-Roleplay#2]

**Comment by ghost** [comment]
Lösung gefunden?

**Comment by Filler-dev** [comment]
> Lösung gefunden?

Nein

**Comment by juztim** [comment]
Ihr könnt zu der Liste vom DbContext bei der jeweiligen Liste n neues Objekt der Types der jeweiligen Liste hinzufügen und es dann speichern mit dbContext.Savechanges(), baut euch n Command oder so dafür bei Bedarf

**Comment by ghost** [comment]
> DbContext

Schon einmal danke für die Antwort. Das Problem, welches ich habe ist, dass ich mir einen Gamemode
zum lernen gesucht habe und auf das hier gestoßen bin. Leider bin ich in der ganzen Sache jetzt nicht so
fit, vorallem hab ich zuvor noch nie mit MongoDB gearbeitet, dass ich das einfach umsetzen könnte, was du da geschrieben hast. Zumal ich ja sogar das Problem habe,
dass der Server läuft, aber der Login-Bildschirm nicht auftaucht, wenn ich connecte. Fehlermeldungen erhalte
ich keine und Clientside ist auch alles eingebunden. Könnte vielleicht auch damit zusammenhängen, dass
er quasi keine Datenbank-Tabellen findet? Danke schonmal.

**Comment by juztim** [comment]
> Könnte vielleicht auch damit zusammenhängen, dass
er quasi keine Datenbank-Tabellen findet?

Tabellen welche beschrieben werden also zum Beispiel Users, werden bei der Registration werden automatisch erstellt.

**Comment by ghost** [comment]
Okay, dann mach ich wohl irgendwo einen anderen Fehler.

**Comment by ghost** [comment]
> Tabellen welche beschrieben werden also zum Beispiel Users, werden bei der Registration werden automatisch erstellt.

Stimmt, danke dafür. Hab das jetzt soweit hinbekommen. Auch die Charakter-Creation funktioniert bis auf eine Sache. Wenn ich fertig bin und auf 'Done' klicke, erhalte ich in VS folgende Meldung zu 
`var json = JsonConvert.DeserializeObject<Chardata>(data);`: System.ArgumentNullException: "Value cannot be null. Arg_ParamName_Name". Weißt du da vielleicht noch eine Lösung? Sitze seit Stunden dran, um eine Lösung zu finden, aber ich weiß nicht mehr weiter. Danke^^

**Comment by juztim** [comment]
Dann musst du clientseitig mal ein wenig loggen wo die Daten "verloren" gehen
